package.json

```diff
@@ -33,57 +32,49 @@
     "index.js.map",
     "index.js"
   ],
-  "pre-commit": [
-    "build",
-    "lint",
-    "test"
-  ],
+  "husky": {
+    "hooks": {
+      "pre-commit": "yarn run build && yarn run lint && yarn run test"
+    }
+  },
   "dependencies": {
     "@rdfjs/types": "*",
-    "@types/readable-stream": "^2.3.11",
-    "@types/sparqljs": "^3.1.3",
-    "abort-controller": "^3.0.0",
-    "cross-fetch": "^3.0.6",
-    "is-stream": "^2.0.0",
-    "minimist": "^1.2.0",
-    "n3": "^1.6.3",
-    "rdf-string": "^1.6.0",
     "@smessie/readable-web-to-node-stream": "^3.0.3",
-    "sparqljs": "^3.1.2",
-    "sparqljson-parse": "^2.2.0",
-    "sparqlxml-parse": "^2.1.1",
-    "stream-to-string": "^1.1.0"
+    "@types/n3": "^1.0.0",
+    "@types/readable-stream": "^4.0.0",
+    "@types/sparqljs": "^3.0.0",
+    "cross-fetch": "^4.0.0",
+    "is-stream": "^2.0.0",
+    "n3": "^1.0.0",
+    "rdf-string": "^1.0.0",
+    "sparqljs": "^3.0.0",
+    "sparqljson-parse": "^2.0.0",
+    "sparqlxml-parse": "^2.0.0",
+    "stream-to-string": "^1.0.0",
+    "yargs": "^17.0.0"
   },
   "devDependencies": {
-    "@types/jest": "^29.5.4",
-    "@types/minimist": "^1.2.0",
-    "@types/n3": "^1.10.3",
+    "@rubensworks/eslint-config": "^2.0.0",
```

Comment:
I released v3 recently, perhaps we can update to that one already (see Comunica next/major if you want an example)?